### ISSUE TITLE
Add google business info to side widget

### ISSUE THREAD

**Issue by deam0n** [searxng/searxng#1925]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be great to search for a company name and get a side "widget" with the opening hours

**Describe alternatives you've considered**
To use google.com

**Additional context**
In our family we use this feature a lot to know if a business is open or not.
Despite all the info google provides the most important is: Working hours / Phone / address
(Don't really care about rating, images, etc...)
![image](https://user-images.githubusercontent.com/1869650/199621219-0f655df3-5e21-4f0f-acd5-f39b9ccfc96f.png)

**Comment by D371L** [comment]
It's a good widget, I don't think it will be related to a problem.

**Comment by deam0n** [comment]
> It's a good widget, I don't think it will be related to a problem.

why would it be related to a problem?

It's a feature request....

**Comment by D371L** [comment]
> > It's a good widget, I don't think it will be related to a problem.
> 
> why would it be related to a problem?
> 
> It's a feature request....

You're right, I didn't get the point at first. Seems to be a good feature.

**Comment by bonswouar** [comment]
I would also love this feature, using maps for business hours & (updated) addresses is basically the one last thing I need (directly) Google for, that's annoying :)
I believe this should be doable with #1642 ? Can't wait to see those features implemented!

**Comment by GuilianCD** [comment]
Couldn't this be possible with / also include data from OSM ? I'm pretty sure wikidata's got some stuff as well.

**Comment by deam0n** [comment]
I guess there's no way to implement this?
